e2e/testcafe-devextreme/tests/editors/checkBox/common.ts

```diff
@@ -47,3 +59,52 @@ fixture.disablePageReloads`CheckBox`
     await createWidget('dxCheckBox', { value: undefined, text: 'indeterminate', rtlEnabled: true }, '#indeterminateRTL');
   });
 });
+
+['one two three', 'label'].forEach((text) => {
+  test(`Checkbox configurations label=${text}`, async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await testScreenshot(t, takeScreenshot, `CheckBox configurations label=${text}.png`, { element: '#container', shouldTestInCompact: true });
+
+    await testScreenshot(t, takeScreenshot, `CheckBox configurations label=${text}.png`, { element: '#container', theme: getFullThemeName().replace('light', 'dark') });
```

Comment:
i'd move getFullThemeName().replace('light', 'dark')  to theme utils, and call the helper function like getDarkTheneName